sway/commands/client.c

```diff
@@ -0,0 +1,79 @@
+#include "log.h"
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/container.h"
+
+static bool parse_color(char *hexstring, float (*dest)[4]) {
+	if (hexstring[0] != '#') {
+		return false;
+	}
+
+	if (strlen(hexstring) != 7) {
+		return false;
+	}
+
+	++hexstring;
+	char *end;
+	uint32_t decimal = strtol(hexstring, &end, 16);
+
+	if (*end != '\0') {
+		return false;
+	}
+
+	(*dest)[0] = ((decimal >> 16) & 0xff) / 255.0;
+	(*dest)[1] = ((decimal >> 8) & 0xff) / 255.0;
+	(*dest)[2] = (decimal & 0xff) / 255.0;
+	(*dest)[3] = 1.0;
```

Comment:
Parse alpha?

Reply:
At the moment I take the alpha from the container (from the `opacity` command). I guess we can allow them to enter the alpha part in their config and combine the two. I'd rather this be a separate PR so this one doesn't become huge.

Reply:
ACK. Please file a follow-up ticket when this PR gets merged.

Reply:
imo we should disregard the client opacity when drawing the borders

Reply:
You mean, the per-view opacity?

Yeah, w/e, I don't know what the use-case for opacity is in the first place.

Reply:
Yeah, we should just use the border's configured alpha color.